storey/redis_driver.py

```diff
@@ -98,19 +93,15 @@ async def inner_fn(*args, **kwargs):
     @property
     def redis(self):
         if self._redis is None:
-            if self._type is RedisType.STANDALONE:
+            try:
+                self._redis = redis.cluster.RedisCluster.from_url(self._redis_url, decode_responses=True)
+            except redis.cluster.RedisClusterException:
                 self._redis = redis.Redis.from_url(self._redis_url, decode_responses=True)
-            else:
-                self._redis = redis.cluster.RedisCluster.from_url(self._redis_url, decode_response=True)
         return self._redis
 
-    async def close(self):
```

Comment:
why did you remove the close()?

Reply:
Closing standalone redis connection in practice does nothing, but closing clustered one closes connections to each of the instances leaving redis object in "zombi" state. Trying reuse the object later causes exception